nixos/modules/services/web-servers/phpfpm/default.nix

```diff
@@ -48,6 +31,83 @@ let
     cat $phpPackage/etc/php.ini $nixDefaultsPath $phpOptionsPath > $out
   '';
 
+  poolOpts = { lib, name, ... }:
+    let
+      poolOpts = cfg.pools."${name}";
+    in
+    {
+      options = rec {
+        socket = mkOption {
+          type = types.str;
+          readOnly = true;
+          description = ''
+            Path to the unix socket file on which to accept FastCGI requests.
+            <note><para>This option is read-only and managed by NixOS.</note></para>
+          '';
+        };
+
+        listen = mkOption {
+          type = types.str;
+          default = "";
+          example = "/path/to/unix/socket";
```

Comment:
Do we need this to be configurable?

Reply:
Nevermind. This was the case before already...

Reply:
See https://github.com/NixOS/nixpkgs/pull/65706#discussion_r309942883